cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesModelExtension.java

```diff
@@ -18,15 +19,21 @@
 public class CgmesModelExtension extends AbstractExtension<Network> {
 
     private final CgmesModel cgmes;
+    private final CgmesUpdate cgmesUpdate;
 
-    public CgmesModelExtension(CgmesModel cgmes) {
+    public CgmesModelExtension(CgmesModel cgmes, CgmesUpdate cgmesUpdate) {
         this.cgmes = Objects.requireNonNull(cgmes);
+        this.cgmesUpdate = Objects.requireNonNull(cgmesUpdate);
     }
 
     public CgmesModel getCgmesModel() {
         return cgmes;
     }
 
+    public CgmesUpdate getCgmesUpdate() {
+        return cgmesUpdate;
+    }
```

Comment:
Checked: it seems CgmesUpdate is never null, you can remove assertions in CgmesExport.

Reply:
ok